uconfig.h

```diff
@@ -1229,8 +1229,8 @@
  *	This symbol contains the ~name expanded version of ARCHLIB, to be used
  *	in programs that are not prepared to deal with ~ expansion at run-time.
  */
-/*#define ARCHLIB "/usr/local/lib/perl5/5.33/unknown"		/ **/
-/*#define ARCHLIB_EXP "/usr/local/lib/perl5/5.33/unknown"		/ **/
+/*#define ARCHLIB "/usr/local/lib/perl/7.0/unknown"		/ **/
+/*#define ARCHLIB_EXP "/usr/local/lib/perl/7.0/unknown"		/ **/
```

Comment:
These (and others in this file) are all "7.0" - should they not be "7.1"?

Reply:
you are right this is fixed by 7b21055d
this will be amended to the 7.1.0 commit